pkg/controller/csi_handler.go

```diff
@@ -743,13 +743,11 @@ func (h *csiHandler) getNodeID(driver string, nodeName string, va *storage.Volum
 			return nodeID, nil
 		}
 		// CSINode exists, but does not have the requested driver.
-		errMessage := fmt.Sprintf("CSINode %s does not contain driver %s", nodeName, driver)
-		klog.V(4).Info(errMessage)
-		return "", errors.New(errMessage)
+		err = errors.New(fmt.Sprintf("CSINode %s does not contain driver %s", nodeName, driver))
```

Comment:
Can you add more comments here to explain how this could happen like what you said in the PR description?

Reply:
done